### PR TITLE
feature: add default flag option in config

### PR DESCRIPTION
**Pull request by stefanasandei**
I added a `default_flag` option in the config, so when you `prideful` without any arguments, it will use that default flag.

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -11,55 +11,47 @@ mod default {
...
     - [ "#770088", 2 ]
-
   pastel:
     - [ "#F59679", 2 ]
     - [ "#F9CE7B", 2 ]
     - [ "#FCF37C", 2 ]
     - [ "#9FFD7C", 2 ]
     - [ "#7BA1FB", 2 ]
     - [ "#AC7CFA", 2 ]
-
```
Comment (angelofallars) [inline src/config.rs]: The whitespace in the default config is necessary for clarity. Can you add them back?

**Review by angelofallars — CHANGES_REQUESTED** [review]

File: src/config.rs (hunk 1 above)
Comment (stefanasandei) [reply src/config.rs]: I'm using vscode with prettier, which automatically removed those whitespaces. Yes I agree they're needed for clarity.

**Comment by angelofallars** [comment]
The CI for this project lints and builds the program for every push or pull request. The lint check unfortunately fails. You should do `cargo fmt --` in the prideful directory to format the Rust code and make the lint check pass. Don't worry, I get a lot of failed checks too in [swhkd](https://waycrate.github.io/swhkd/). 😅

image

**Comment by angelofallars** [comment]
Looks good to me! Thanks for contributing. I will merge this into the dev branch now and integrate it in the next release, along with more config options.

Wanna join a community with some Rust devs btw? Join the [Waycrate](https://waycrate.github.io/index.html) Discord server [here](https://discord.gg/KKZRDYrRYW).

**Review by angelofallars — APPROVED** [review]

**Comment by stefanasandei** [comment]
Thanks! I will join the discord server.

**Pull request merged**